elide-annotations/src/main/java/com/yahoo/elide/security/RequestScope.java

```diff
@@ -1,16 +1,13 @@
 /*
- * Copyright 2016, Yahoo Inc.
+ * Copyright 2020, Yahoo Inc.
  * Licensed under the Apache License, Version 2.0
  * See LICENSE file in project root for terms.
  */
 package com.yahoo.elide.security;
 
-import java.util.Set;
-
 /**
  * The request scope interface passed to checks.
  */
 public interface RequestScope {
     User getUser();
```

Comment:
@aklish Is this all that is left in RequestScope?

Reply:
For now - yes.